contrib/build-wine/deterministic.spec

```diff
@@ -50,6 +52,7 @@ datas = [
     (home+'lib/locale', 'electroncash/locale'),
     (home+'gui/qt/data/ecsupplemental_win.ttf', 'electroncash_gui/qt/data'),
     (home+'plugins', 'electroncash_plugins'),
+    (PYHOME+'\\Lib\\site-packages\\smartcard\\scard\\_scard.cp36-win32.pyd', '.\\smartcard\\scard\\'), #Satochip
```

Comment:
Is this needed? PyInstaller doesn't pick it up huh?

Reply:
The maintainer of the project does not provide official releases for Windows 32-bit apparently. Since the project is open-source, it should be possible to build it from sources but it looks quite complex (LudovicRousseau/pyscard#55). The link provided in the script comes from the appveyor account for pyscard (https://ci.appveyor.com/project/LudovicRousseau/pyscard), and is the suggested workaround provided by the maintainer (see LudovicRousseau/pyscard#55 (comment)). Normally, it should be more straightforward for linux/mac.